src/objects/page/page.scss

```diff
@@ -18,12 +18,19 @@
  * These classes aren't strictly required, but they keep the HTML source a
  * little easier to read while also supporting swapping of source order if
  * that's ever something that content calls for.
+ *
+ * 1. By default grid items have `min-width:auto` which is breaking layout on
+ *    blog posts with long code samples. Setting `min-width:0` allows the grid
+ *    items to shrink, regardless of their intrinsic size.
+ *    @see https://ishadeed.com/article/min-content-size-css-grid/
  */
 
 .o-page__content {
   grid-row: 1;
+  min-width: 0; // 1
 }
 
 .o-page__footer {
   grid-row: 2;
+  min-width: 0; // 1
```

Comment:
Out of curiosity, would setting `grid-template-columns: minmax(0, 1fr);` on the `o-page` element have the same effect as setting `min-width: 0` on the child elements?

https://css-tricks.com/you-want-minmax10px-1fr-not-1fr/

Reply:
FWIW, I tried out @tylersticka's suggestion in the browser via DevTools and it seems to work. I'm am going to read the article to better understand why. 📖 🔍 👀

Reply:
Okay, I'm convinced in that we should explore `grid-template-columns: minmax(0, 1fr);` as @tylersticka suggested! 🙂 

This article helped explain a bit more for me (linked in the original shared article): https://css-tricks.com/preventing-a-grid-blowout/

Reply:
Good call, thanks @tylersticka!

Updated.